.dockerignore

```diff
@@ -22,6 +22,12 @@ packages/stat-logger
 **/_agstate
 .vagrant
 endo-sha.txt
+# We avoid copying these into a docker build context, because we're
```

Comment:
Seems odd to make the build fetch the submodule when it's already on the source disk. Now that we have the environment flag for being in Docker, doesn't that suffice for the conditional logic?

Reply:
Yeah, going to try omitting the `.git` subdir instead. That probably obviates the need for the environment variable too.